internal/metarepos/raft_metadata_repository.go

```diff
@@ -963,6 +963,9 @@ func (mr *RaftMetadataRepository) applyCommit(r *mrpb.Commit, appliedIndex uint6
 			mr.storage.TrimLogStreamCommitHistory(trimVer) //nolint:errcheck,revive // TODO:: Handle an error returned.
 		}
 
+		mr.tmStub.mb.Gauges("mr.commit.history.count").Record(ctx,
+			int64(mr.storage.getLastCommitResultsNoLock().GetVersion())-int64(mr.storage.getFirstCommitResultsNoLock().GetVersion()))
```

Comment:
I'm not certain, but isn't it necessary to add one to the calculation of the number of commit history, which is `last - first + 1`?

Reply:
fixed it